com.spotify.Client.json

```diff
@@ -150,6 +150,88 @@
                 }
             ]
         },
+        /* THESE CANNOT BE UPDATED UNTIL SPOTIFY SUPPORTS ffmpeg 5 ------------------------------------------------------- */
+        {
+            "name": "sdl2",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=/app -B Build",
+                "cd Build && make -j${FLATPAK_BUILDER_N_JOBS:-1} && make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libsdl-org/SDL/releases/download/release-2.0.22/SDL2-2.0.22.tar.gz",
+                    "sha256": "fe7cbf3127882e3fc7259a75a0cb585620272c51745d3852ab9dd87960697f2e"
+                }
+            ]
+        },
+        {
+            "name": "ffmpeg-4.4",
+            "build-options": {
+                "cflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+                "cxxflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+                "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now",
+                "prefix": "/app/",
```

Comment:
Isn't it the default?

Reply:
No for some reason when the prefix was left out ffmpeg tried installing in the wrong place

Reply:
That.s weird since it wasn't needed before: https://github.com/flathub/com.spotify.Client/commit/554b766ee55434d2eb35612e674d0b1dfce3ac9f

Reply:
ill test locally to be sure and update

Reply:
Yeah i have remove the /app/ and you appear to right. strange. Ill just double check and then commit the change.